x/sequencer/keeper/rotation.go

```diff
@@ -46,19 +46,24 @@ func (k Keeper) ChooseSuccessorForFinishedNotices(ctx sdk.Context, now time.Time
 	}
 	for _, seq := range seqs {
 		k.removeFromNoticeQueue(ctx, seq)
-		if err := k.setSuccessorForRotatingRollapp(ctx, seq.RollappId); err != nil {
-			return errorsmod.Wrap(err, "choose successor")
+
+		// Very important to check, because we do not explicitly remove sequencers from the queue: if they are forcibly
```

Comment:
might be cleaner to just `k.removeFromNoticeQueue(ctx, seq)` when proposer forcibly rotated